controllers/resetPassword.js

```diff
@@ -0,0 +1,26 @@
+const User = require('../db/models/User')
+
+const jwt = require('jsonwebtoken')
+const encrypt = require('./encrypt')
+
+exports.receiveNewPassword = (req, res) => {
+  const { userId, token } = req.params
+  //const { password, userId, token } = req.body
```

Comment:
?

Reply:
removed

Reply:
and changed to req.body which in comment